pkg/agent/flowexporter/types.go

```diff
@@ -70,4 +70,6 @@ type FlowRecord struct {
 	PrevBytes          uint64
 	PrevReversePackets uint64
 	PrevReverseBytes   uint64
+	V4Enabled          bool
```

Comment:
Just need one field isIPv6 as one flow record corresponds to either v4 or v6.

Reply:
Updated.